airflow/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Added Redhat-UBI-based image ([#97]).
```

Comment:
Isn't this more of a changed entry?

Reply:
This change will result in images that are named differently (the python variable is used for dnf installs rather than locating the airflow image), so I will delete the old ones when the build action has been done. But, yes, it is essentially a change: I'll edit that.